.env.io-onboarding-pa-mock-backend.development

```
@@ -2,3 +2,4 @@ IO_ONBOARDING_PA_API_HOST=http://localhost
 IO_ONBOARDING_PA_API_PORT=3000
 IO_ONBOARDING_PA_SHOW_FAKE_IDP=1
 IO_ONBOARDING_PA_IS_MOCK_ENV=1
+IO_ONBOARDING_PA_SESSION_TOKEN_DOMAIN="localhost"
```

```suggestion
IO_ONBOARDING_PA_SESSION_TOKEN_DOMAIN=localhost
```

docker-compose does not like double quotes around env values